runtime/src/main/kotlin/kotlin/native/internal/GC.kt

```diff
@@ -78,6 +78,19 @@ object GC {
         get() = getTuneThreshold()
         set(value) = setTuneThreshold(value)
 
+    /**
+     * Detect cyclic references going via atomic references and return list of cycle-inducing objects
+     * or `null` if the leak detector is not available.
```

Comment:
So a user getting `null` from this function would have no idea on which is the case: no leaks or unavailable leak detector.

Reply:
Added reference to appropriate docs.